utils/_context/_scenarios.py

```diff
@@ -850,6 +850,8 @@ class scenarios:
 
     otel_tracing_e2e = OpenTelemetryScenario("OTEL_TRACING_E2E")
 
+    apm_tracing_e2e_otel_span = EndToEndScenario("APM_TRACING_E2E_OTEL_SPAN")
```

Comment:
`EndToEndScenario` spwans weblog with datadog tracers, not open telemetry tracer. Or did I missed something ?

Reply:
I was following the suite of ``` # APM tracing end-to-end scenarios```
It is an e2e test of Datadog Otel API wrapper